go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rook/rook
 
 go 1.19
 
+replace github.com/rook/rook/pkg/apis => ./pkg/apis
```

Comment:
I've noticed that many projects use `pkg/api` instead of `pkg/apis`. Is there a plan to continue using `api` instead of `apis`?

Reply:
I would not want to change the project layout with this PR.